### PR TITLE
#28 Support multiple languages

### PR DESCRIPTION
**Pull request by Muhet**
#### What does this PR do?
International languages translation
#### Description of Task to be completed?
 This PR will help User to change from the default language to the desired languages
#### How should this be manually tested?
To test it, where the return is use t(keyword)

#### Questions:

### CODE REVIEW

File: locales/sp/translation.json
Comment (mukunzidd) [inline locales/sp/translation.json:1]: Will people know how the words in Spanish? Did you agree with the team on the languages?

File: locales/ch/translation.json
Comment (mukunzidd) [inline locales/ch/translation.json:1]: Isn't Chinese/Cantonese a stretch? @Muhet talk to the team first.

File: package.json
Comment (mukunzidd) [inline package.json:1]: Wait and update your package.json file with the latest changes since the project setup also added it.

File: server.js
Comment (mukunzidd) [inline server.js:1]: isn't there a better folder or files to configure this middleware outside of the entry point of the app. Like a middleware folder. Please check the project structure.

**Review by mukunzidd — CHANGES_REQUESTED** [review]

**Comment by mukunzidd** [comment]
@Muhet also remove the square. brackets around the card number and the rest of the title.

File: server.js
Comment (Muhet) [reply server.js:1]: okey i'll look into it

File: locales/sp/translation.json
Comment (Muhet) [reply locales/sp/translation.json:1]: > Will people know how the words in Spanish? Did you agree with the team on the languages?

I changed it. I remain with two. French and English

File: locales/ch/translation.json
Comment (Muhet) [reply locales/ch/translation.json:1]: We had a conversation and we agreed that we will use french and english

**Review by chanceDira — COMMENTED** [review]
Nice job @Muhet,
Work on these changes requested and resolve conflict, then we are good

File: server.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const Backend = require('i18next-fs-backend');
+const middleware = require('i18next-http-middleware');
+
+i18next.use(Backend)
+.use(middleware.LanguageDetector)
+.init({
+    fallbackLng: 'en',
+    backend:{
+        loadPath: './locales/{{lng}}/translation.json'
+    }
```
Comment (ALVINdimpos) [inline server.js]: great job man

Comment (Muhet) [reply server.js]: thx

**Review by ALVINdimpos — COMMENTED** [review]
cul

**Review by ALVINdimpos — APPROVED** [review]

**Review by bryan936M — APPROVED** [review]
Congratulations on the successful completion of your task

**Review by faid-terence — APPROVED** [review]
Well done @Muhet ,

**Review by NTElissa — APPROVED** [review]

**Review by chanceDira — APPROVED** [review]
Good work @Muhet

**Review by vanessa-abewe — APPROVED** [review]

**Review by yobuir — APPROVED** [review]
@Muhet nice work

**Review by mukunzidd — APPROVED** [review]

**Review by kagaba-etienne — APPROVED** [review]
Good Job!!!

**Pull request merged**